glusterd2/commands/volumes/optiongroup-delete.go

```diff
@@ -0,0 +1,55 @@
+package volumecommands
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/pkg/api"
+
+	"github.com/gorilla/mux"
+)
+
+func optionGroupDeleteHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	groupName := mux.Vars(r)["groupname"]
+
```

Comment:
We may need a lock between `Store.Get` here and `Store.Put` below. For now, this is fine.